markdown/blockparser.py

```diff
@@ -62,14 +66,21 @@ class BlockParser:
 
     A wrapper class that stitches the various `BlockProcessors` together,
     looping through them and creating an `ElementTree` object.
+
```

Comment:
No strong  feeling about this. We should just try and be consistent. Or not, it's not super important.

Reply:
Sorry, I'm not sure what you are referring to here.

Reply:
I meant, whether to keep trailing new lines in docstrings.

Currently it seems the project follows these rules:

- Leading and trailing space around summary.
- Trailing blank line after body.